src/test/kotlin/uk/gov/justice/hmpps/offenderevents/e2e/HMPPSDomainEventsIntTest.kt

```diff
@@ -131,59 +114,22 @@ class HMPPSDomainEventsIntTest : QueueListenerIntegrationTest() {
         await().until { getNumberOfMessagesCurrentlyOnHMPPSEventTestQueue() == 1 }
       }
 
-      @Test
```

Comment:
This test seems redundant it seems to be saying "when I publish a NOMIS prison event I receive a NOMIS prison event" - we are of course only interested in what domain events get fired